site/en/external/registry.md

```diff
@@ -72,6 +72,17 @@ An index registry must follow the format below:
                 By default, the archive type is determined from the file extension of the URL. If the file has
                 no extension, you can explicitly specify one of the following: `"zip"`, `"jar"`, `"war"`, `"aar"`,
                 `"tar"`, `"tar.gz"`, `"tgz"`, `"tar.xz"`, `"txz"`, `"tar.zst"`, `"tzst"`, `tar.bz2`, `"ar"`, or `"deb"`.
+        *   The type can be changed to use a git repository, with these fields:
+            *   `type`: `git_repository`
+            *   The following fields as described at https://bazel.build/rules/lib/repo/git:
+                * `remote`
+                * `commit`
+                * `shallow_since`
+                * `tag`
+                * `init_submodules`
+                * `recursive_init_submodules`
```

Comment:
also remember to remove this (given the discussion in the other thread)

Reply:
Done.